contracts/package.json

```diff
@@ -44,5 +44,8 @@
     "ts-node": "^10.0.0",
     "typechain": "^5.0.0",
     "typescript": "^4.3.2"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.3.3"
```

Comment:
can we move this to `devDependencies` with the others?

Reply:
done!